File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int age) {
    string answer = "";
    string lang = "abcdefghij";
    string stAge = to_string(age);
    for(int i=0;i<stAge.size();i++)
    {
        answer += lang[stAge[i]-'0'];
    }
    return answer;
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cs
using System;
using System.Text;

public class Solution {
    public string solution(int age) {
        StringBuilder answer = new StringBuilder("", 3);
        string lang = "abcdefghij";
        string stAge = Convert.ToString(age);
        for(int i=0;i<stAge.Length;i++)
        {
            answer.Append(lang[stAge[i] - '0']);
        }
        return answer.ToString();
    }
}